config/configuration.yaml.example

```diff
@@ -96,27 +103,23 @@ browser:
 keyboard:
 
 automation:
```

Comment:
This example is actually completely wrong, it's in the old config format. No mention of triggers or actions.